packages/type/src/type-serialization.ts

```diff
@@ -347,7 +353,7 @@ function serialize(type: Type, state: SerializerState): SerializedTypeReference
             }
 
             const classType = getClassName(type.classType);
-            const globalObject: boolean = global && (global as any)[classType] === type.classType;
+            const globalObject: boolean = envGlobal && (envGlobal as any)[classType] === type.classType;
```

Comment:
can we not just define `const envGlobal: any = ...` and then omit this type cast?